cluster/images/hyperkube/master.json

```diff
@@ -23,6 +23,7 @@
               "apiserver",
               "--portal-net=10.0.0.1/24",
               "--address=127.0.0.1",
+              "--read-write-port=8080",
```

Comment:
again, TLS should not equal authentication. Is the issue that TLS is not enabled, or that no authentication method is set up for the TLS endpoint? Making it non-deterministic whether the kubernetes service is TLS or not makes it very hard to write containers that make use of it